bin/ck8s

```diff
@@ -14,7 +14,7 @@ usage() {
     echo "  bootstrap <wc|sc>                                 bootstrap the cluster" 1>&2
     echo "  apps <wc|sc> [--sync] [--skip-template-validate]  deploy the applications" 1>&2
     echo "  apply <wc|sc> [--sync] [--skip-template-validate] bootstrap and apps" 1>&2
-    echo "  test <wc|sc> [--logging-enabled]                  test the applications" 1>&2
+    echo "  test <wc|sc>                                      test the applications" 1>&2
```

Comment:
Why not keep `[--logging-enabled]`?

Reply:
I guess it would be nice if you could still use `--logging-enabled` when running all tests, even though it would only be used in the `test sc/wc apps` part